app/serializers/rent_serializer.rb

```diff
@@ -0,0 +1,3 @@
+class RentSerializer < ActiveModel::Serializer
+  attributes :user, :book, :from, :to
```

Comment:
`:user` and `:book` are relations, you have to use ActiveModelSerializer's relations to display it correctly. [Here](https://github.com/rails-api/active_model_serializers/blob/v0.10.6/docs/general/serializers.md#associations) is the documentation about this.